Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -938,6 +938,18 @@ theorem lintegral_pos_iff_support {f : α → ℝ≥0∞} (hf : Measurable f) :
   simp [pos_iff_ne_zero, hf, Filter.EventuallyEq, ae_iff, Function.support]
 #align measure_theory.lintegral_pos_iff_support MeasureTheory.lintegral_pos_iff_support
 
+theorem lintegral_positive {C : Set α} (hmC : MeasurableSet C)
+    (h_nonneg : 0 < μ C) {h : α → ℝ≥0∞} (h_neq : ∀ x ∈ C, h x ≠ 0) (hm : Measurable h) :
+    0 < ∫⁻ x in C, h x ∂μ := by
```

Comment:
Please prove an `iff` version (probably, with `∃ᵐ x ∂μ, 0 < h x` in the RHS) using `lintegral_eq_zero_iff` and `pos_iff_ne_zero`.

Reply:
Ok thanks for your comment and indications. See [#cbf6aa4](https://github.com/leanprover-community/mathlib4/pull/13172/commits/cbf6aa468599f981d937bc3992d209eba8d6647f)